db/migrate/20201021152850_add_index_to_statistics_announcement_publication_id.rb

```diff
@@ -0,0 +1,8 @@
+class AddIndexToStatisticsAnnouncementPublicationId < ActiveRecord::Migration[5.1]
+  def change
+    change_table :statistics_announcements do |t|
```

Comment:
Sorry, I forgot this:

```suggestion
    change_table :statistics_announcements, bulk: true do |t|
```